src/foam/gui/AppStyles.js

```diff
@@ -0,0 +1,81 @@
+
+foam.CLASS({
+  package: 'foam.gui',
```

Comment:
Why the 'gui' package instead of 'u2', where all of the other FOAM UI code is?

Reply:
it's up to you.

Reply:
I figured these views are a little more detailed comparing to the other in u2

Reply:
I would move to u2 except for the parts related to to controller, which should move to the same package in nanos where the controller is.